domain-story-modeler-thesis/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Check if the VITE_BACKEND_URL environment variable is set
+if [ -z "$VITE_BACKEND_URL" ]; then
+  echo "Error: 
+       You must specify VITE_BACKEND_URL to a valid URL.
```

Comment:
The error message is misleading: While a valid URL is, of course, desired, the check is basically whether the variable is empty. A user could think that the problem is the validity of the URL and not that the input is empty. Consider sth like this "VITE_BACKEND_URL must not be empty. Please enter a valid URL, for example: ...".